packages/expo-contacts/android/src/main/java/expo/modules/contacts/Contact.kt

```diff
@@ -424,8 +422,10 @@ class Contact(var contactId: String) {
         try {
           hasYear = !dateString!!.startsWith("--")
           if (hasYear) {
+            val datePattern = SimpleDateFormat("yyyy-MM-dd", Locale.US)
```

Comment:
nothing wrong with `Locale.getDefault()` in this case, but I pinned it to `Locale.US` to make it "raise no questions" if that makes sense.